patches/minecraft/net/minecraft/item/ItemGroup.java.patch

```diff
@@ -9,7 +9,15 @@
     public static final ItemGroup field_78030_b = (new ItemGroup(0, "buildingBlocks") {
        @OnlyIn(Dist.CLIENT)
        public ItemStack func_78016_d() {
-@@ -104,7 +104,16 @@
+@@ -95,6 +95,7 @@
+          return new ItemStack(Blocks.field_150486_ae);
+       }
+    }).func_78025_a("inventory.png").func_78022_j().func_78014_h();
++   private static final net.minecraft.util.ResourceLocation CREATIVE_INVENTORY_TABS = new net.minecraft.util.ResourceLocation("textures/gui/container/creative_inventory/tabs.png");
```

Comment:
put patched-in fields near the methods they're used (put it above getTabsImage in this case)